docs/source/developer_notes/layouts.rst

```diff
@@ -0,0 +1,79 @@
+Layouts
+=======
+
+PlotArea
+--------
+
+This is the main base class within layouts. A ``Figure`` and ``Dock`` are areas within a ``Subplot`` that
+inherit from ``PlotArea``.
+
+``PlotArea`` has the following key properties that allow it to be a "plot area" that can be used to view graphical objects:
+
+* scene - instance of ``pygfx.Scene``
+* canvas - instance of ``WgpuCanvas``
+* renderer - instance of ``pygfx.WgpuRenderer``
+* viewport - instance of ``pygfx.Viewport``
+* camera - instance of ``pygfx.PerspectiveCamera``, we always just use ``PerspectiveCamera`` and just set ``camera.fov = 0`` for orthographic projections
+* controller - instance of ``pygfx.Controller``
+
```

Comment:
add something like: "If these concepts are unfamiliar to you we recommend learning about how rendering engines work, the pygfx guide is a great place to start! https://docs.pygfx.org/stable/guide.html"

Reply:
This is also a short quick video that goes through the basic concepts: https://www.youtube.com/watch?v=cvcAjgMUPUA